src/app/introduction/activity-1/activity-1.component.ts

```diff
@@ -23,8 +24,9 @@ export class HighlightActivityComponent implements OnInit {
     public language: any;
     public alerts: any;
     public completed = false;
+    private readonly currStage = 1;
```

Comment:
Constant naming convention to be followed, please change it in the other files as well.